README.md

```diff
@@ -1,17 +1,79 @@
-## Documentation Guidelines
+# Documentation Guidelines
 
-### 🚀 Contributing
+## 🚀 Contributing
 
-### Before Proposing Additions
+## Before Proposing Additions
 
-### Style Guidelines
+## Style Guidelines
 
-#### Code Examples
+### Code Examples
 
-#### Notices
+### Notices
 
-#### Screenshots
+### Screenshots
 
-#### Terminology
+### Terminology
+Appwrite has many services and features with many ways to express the same concepts. To avoid confusion, this section suggests a standard set of terms used across Appwrite documentation to describe features and concepts.
+#### General
+| **Term**      | **Suggested Usage.**                                                                                                                                                                                              |
+|---------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Appwrite      | Written as "Appwrite" not "AppWrite" or "appwrite".                      |
+| Projects      | Each Appwrite instance can have many Appwrite Projects. Use "Appwrite Projects" as a proper noun when referring to the concept, use project or projects when referring to specific projects.                      |
+| Console       | Refers to the the GUI. Can also be referred to as the "Appwrite Console" to distinguish from a Mac, Linux, or Windows machine's console.                                                                          |
+| Console Users | Refers to users that are registered to have access to the Appwrite Console. Not a proper noun, use as "a console user" or "the console user". Differenciate this clearly from users created through a Client SDK. |
+| Client SDK    | Refers to SDKs used by Web, Flutter, Android, and Apple applications. Use as a proper noun, use "a Client SDK" or "the Client SDKs", not client-side SDKs.                                                        |
+| Server SDK    | Refers to SDKs used by backend languages like Java, Node.js, or PHP. Use as a proper noun, use "a Server SDK" or "the Server SDKs", not server-side SDKs.                                                         |
+| Developer     | Refers to Appwrite developers that use Appwrite to create applications.                                                                                                                                           |
+| End User      | Refers to end users of applications with an Appwrite backend. This doesn't refer to developers that interact with Appwrite directly.   |
+| Appwrite Instance | Refers to a single, self-hosted deployment of Appwrite   |
```

Comment:
## Need Feedback

Do we want to call it a deployment of Appwrite, an instance of Appwrite? What language makes the most sense?